integration_tests/mockApis/licence.ts

```diff
@@ -1545,12 +1545,12 @@ export default {
             prisonId: 'BAI',
             sentenceStartDate: '2017-03-01',
             releaseDate: '2024-07-19',
-            confirmedReleaseDate: '2022-11-20',
+            confirmedReleaseDate: '2022-11-10',
             sentenceExpiryDate: '2028-08-31',
             licenceExpiryDate: '2028-08-31',
-            conditionalReleaseDate: '2022-11-21',
+            conditionalReleaseDate: '2022-11-10',
```

Comment:
Changed these dates as the previous values were making the licence appear as being eligible for an early release. This caused a different confirm-create screen to appear, which in turn meant that the createLicence integration test had to click one fewer buttons to progress through certain pages, which caused a break in the tests.